src/extensions/extension-compatibility.ts

```diff
@@ -31,3 +31,7 @@ export function isCompatibleExtension(manifest: LensExtensionManifest): boolean
 
   return false;
 }
+
+export function isCompatibleBundledExtension(manifest: LensExtensionManifest): boolean {
+  return manifest.version === appSemVer.raw;
```

Comment:
Does this work when `appSemVer` has a prerelease tag? (Does `manifest.version` have the matching tag, at least for bundled v5 extensions?)

Reply:
Yes do (I believe). We set them to the exact version that the `package.json` was changed to

Reply:
During build the app version is synced to the bundled extensions: https://github.com/lensapp/lens/blob/master/build/set_build_version.ts#L45